src/main/java/com/bulkbuy/enterprise/controller/NavigationController.java

```diff
@@ -39,19 +42,18 @@ public String getOrderLookup(Model model){
         return "orderLookup";
     }
 
-    @PostMapping(value="/saveOrder", consumes = "application/json", produces = "application/json")
-    public Order saveOrder(@RequestBody Order order) throws Exception {
+    @PostMapping(value="/saveOrder")
+    public String saveOrder(@ModelAttribute("saveOrder") Order savedOrder, Model model) {
```

Comment:
Removed the consumes/produces Json as this throws an exception since the media doesn't match the types being produced or consumed. Added the @ModelAttribute to pull in the data from the fields in the form.